.github/workflows/presubmit.yml

```diff
@@ -65,6 +66,12 @@ jobs:
         id: changed-files
         uses: androidx/changed-files-action@main
 
+      - name: "Get affected files in push or pull_request"
```

Comment:
Btw, we could also just output both from the action although I think the repo checkout for the action should be cached so no big deal.

Reply:
yea idk why i didn't think of that 🤦 .

Reply:
sent a PR, will update this after that one gets merged.